sql/item_furnishing.sql

```diff
@@ -517,3 +517,14 @@ INSERT INTO `item_furnishing` VALUES (3737, 'doll_stand', 2, 2855, 6, 15);
 INSERT INTO `item_furnishing` VALUES (3738, 'eastern_umbrella', 1, 521, 5, 2); -- Moghancement: Gardening - From https://ffxiclopedia.fandom.com/wiki/Eastern_Umbrella (BG doesn't show Moghancement)
 INSERT INTO `item_furnishing` VALUES (3739, 'autumn_tree', 1, 515, 3, 5);
 -- INSERT INTO `item_furnishing` VALUES (3740, 'model_synergy_furnace', 8, 0, 7, 11); -- Moghancement: Synergy Skill Gains does not yet exist, so commenting this row out for now
+-- INSERT INTO `item_furnishing` VALUES (3742, 'painting_of_a_mercenary', 1, 515, 3, 1); -- Moghancement: Mandragora Mania does not yet exist
+INSERT INTO `item_furnishing` VALUES (3743, 'moogle_bed', 1, 520, 6, 8);
```

Comment:
Leaving a comment to link this to #831 so I don't forget about this new furniture.